src/views/GameDetails/GameServer/GameServer.scss

```diff
@@ -0,0 +1,101 @@
+@import "../../../index";
+
+.template-title {
+    color: $color-white;
+    align-self: center;
```

Comment:
Since `align-self` is a property of a flex box, it doesn't actually make any difference here. It's still centered without this line.

Reply:
Changed it and nothing effect. Removed